flexibleengine/resource_flexibleengine_cce_node_v3.go

```diff
@@ -461,6 +476,10 @@ func resourceCCENodeV3Read(d *schema.ResourceData, meta interface{}) error {
 	d.Set("region", GetRegion(d, config))
 	d.Set("private_ip", s.Status.PrivateIP)
 	d.Set("public_ip", s.Status.PublicIP)
+	d.Set("iptype", s.Spec.PublicIP.Eip.IpType)
+	d.Set("bandwidth_charge_mode", s.Spec.PublicIP.Eip.Bandwidth.ChargeMode)
+	d.Set("bandwidth_size", s.Spec.PublicIP.Eip.Bandwidth.Size)
+	d.Set("sharetype", s.Spec.PublicIP.Eip.Bandwidth.ShareType)
```

Comment:
ditto.
These attributes will not included in API response, this empty values setting will cause problems.

Reply:
i removed it also. Thanks again for your review